doc/Configuration.md

```diff
@@ -54,26 +54,16 @@ Here you specify what you want to build. See example configs for sensible
 lists. The order of components is important - it should reflect build
 dependencies, otherwise build would fail.
 
-### CC_`component`
-> Default: Value set in Makefile of the component
+### ENV_`component`
+> Default: None
 
-Custom component specific compiler
+Specify environment variables for component build
 
-Example:
-CC_gui-daemon=clang
-
-### CFLAGS_`component`
-> Default: Value set in Makefile of the component
-
-Custom component specific compiler flags
-
-Example:
-CFLAGS_gui-daemon=-fstack-protector-strong
-
-### LDFLAGS_`component`
-> Default: Value set in Makefile of the component
+For example:
+ENV_gui-daemon = "\
```

Comment:
Many applications (including shell) does not like `-` in env variable name. This is why in this type of settings we replace `-` with `_` (see BRANCH_*, GIT_URL_* etc).